docs/analytics/insights.md

```diff
@@ -13,16 +13,10 @@ cd configuration/playbooks
 Playbooks
 ---------
 
-Create an ansible playbook to deploy Insights and the Analytics API using [`edx/configuration`
-roles](https://github.com/edx/configuration/playbooks/roles).  See
-[`analytics_sandbox.yml`](resources/playbooks/analytics_sandbox.yml) for an example playbook.
-
-If you are splitting these services across multiple EC2 instances, then you'll need two playbooks, e.g.
-[`analytics_insights.yml`](resources/playbooks/analytics_insights.yml) and
-[`analytics_api.yml`](resources/playbooks/analytics_api.yml).
-
-If you're planning to expose the `analytics_api` via nginx, you'll also need this
```

Comment:
Maybe leave a note saying that this template will get installed by running the `analytics_single` playbook?

Reply:
I don't think we need to leave anything in the docs about this.. there are templates all through the `edx:configuration` code which help us out, and this was only mentioned here because we had to manually add it in.